templates/app.yaml

```diff
@@ -246,6 +246,10 @@ Resources:
         - Namespace: 'aws:elasticbeanstalk:application:environment'
           OptionName: TEAM_TO_ROLE_ARN_MAP
           Value: "ssm::/service-catalog/TeamToRoleArnMap"
+        - Namespace: 'aws:elasticbeanstalk:application:environment'
+          OptionName: MARKETPLACE_ID_DYNAMO_TABLE_NAME
+          Value: !ImportValue
+                 'Fn::Sub': '${AWS::Region}-${AWS::StackName}-MarketplaceDynamoDBTable'
```

Comment:
this should be `'Fn::Sub': ${AWS::Region}-marketplace-dynamo-MarketplaceDynamoDBTable` because you exported `MarketplaceDynamoDBTable` in PR https://github.com/Sage-Bionetworks/scipool-infra/pull/214